Gems/GeoJSONSpawner/Code/Source/GeoJSONSpawner/ROS2Interface/GeoJSONSpawnerROS2InterfaceEditorComponent.cpp

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (C) Robotec AI - All Rights Reserved
+ *
+ * This source code is protected under international copyright law.  All rights
+ * reserved and protected by the copyright holders.
+ * This file is confidential and only available to authorized individuals with the
+ * permission of the copyright holders.  If you encounter this file and do not have
+ * permission, please contact the copyright holders and delete this file.
+ */
+
+#include "GeoJSONSpawnerROS2InterfaceEditorComponent.h"
+
+#include "AzCore/Serialization/EditContext.h"
+
+namespace GeoJSONSpawner::ROS2Interface
+{
+    inline constexpr const char* stringMessageType = "std_msgs::msg::String";
+    inline constexpr const char* emptyMessageType = "std_msgs::msg::Empty";
+
+    GeoJSONSpawnerROS2InterfaceEditorComponent::GeoJSONSpawnerROS2InterfaceEditorComponent()
+    {
+        m_configuration.m_spawnTopicConfiguration.m_topic = "geojson/spawn";
+        m_configuration.m_spawnTopicConfiguration.m_type = stringMessageType;
+
+        m_configuration.m_modifyTopicConfiguration.m_topic = "geojson/modify";
+        m_configuration.m_modifyTopicConfiguration.m_type = stringMessageType;
+
+        m_configuration.m_deleteByIdTopicConfiguration.m_topic = "geojson/delete_by_id";
+        m_configuration.m_deleteByIdTopicConfiguration.m_type = stringMessageType;
+
+        m_configuration.m_deleteAllTopicConfiguration.m_topic = "geojson/delete_all";
+        m_configuration.m_deleteAllTopicConfiguration.m_type = emptyMessageType;
+    }
```

Comment:
I think that placing the default topics in the EditorComponent interface is the wrong place. A better place would be the default constructor for the configuration.

This approach probably will always work, but I can imagine a scenario when this function is never called (for example in the game mode), ane there will be no default topics.